lib/src/pull_down_button.dart

```diff
@@ -385,10 +385,18 @@ class _PullDownButtonState extends State<PullDownButton> {
   PullDownButtonAnimationState state = PullDownButtonAnimationState.closed;
 
   Future<void> showButtonMenu() async {
-    var button = context.getRect;
+    final navigator = Navigator.of(
+      context,
+      rootNavigator: widget.useRootNavigator,
+    );
+
+    final overlay = navigator.overlay!.context.findRenderObject()! as RenderBox;
```

Comment:
Nit: the extension for this already exists, might as well use it 😃 

```suggestion
    final overlay = navigator.overlay!.context.currentRenderBox;
```

Reply:
should be addressed in 3afecd3